tests/functional/modules/test_zos_copy_func.py

```diff
@@ -1015,6 +1015,62 @@ def test_copy_non_existent_file_fails(ansible_zos_module, is_remote):
         assert "does not exist" in result.get("msg")
 
 
+@pytest.mark.uss
+@pytest.mark.parametrize("src", [
+    dict(src="/etc/profile", is_file=False, is_binary=False, is_remote=False),
+    dict(src="/etc/profile", is_file=False, is_binary=False, is_remote=True),])
+def test_copy_dest_directory_remain(ansible_zos_module, src):
+    hosts = ansible_zos_module
+    dest_path = "/tmp/test/"
+    try:
+        hosts.all.shell(cmd="mkdir -p {0}".format(dest_path))
```

Comment:
We should do things the same way as in other test cases, like creating folders, if you look at other test cases the preferred way to create a folder in the remote system is:

hosts.all.file(path=dest_path, state="directory")